packages/node-experimental/package.json

```diff
@@ -24,26 +24,26 @@
   },
   "dependencies": {
     "@opentelemetry/api": "~1.6.0",
-    "@opentelemetry/core": "~1.17.0",
-    "@opentelemetry/context-async-hooks": "~1.17.0",
-    "@opentelemetry/instrumentation": "~0.43.0",
-    "@opentelemetry/instrumentation-express": "~0.33.1",
-    "@opentelemetry/instrumentation-fastify": "~0.32.3",
-    "@opentelemetry/instrumentation-graphql": "~0.35.1",
-    "@opentelemetry/instrumentation-http": "~0.43.0",
-    "@opentelemetry/instrumentation-mongodb": "~0.37.0",
-    "@opentelemetry/instrumentation-mongoose": "~0.33.1",
-    "@opentelemetry/instrumentation-mysql": "~0.34.1",
-    "@opentelemetry/instrumentation-mysql2": "~0.34.1",
-    "@opentelemetry/instrumentation-nestjs-core": "~0.33.1",
-    "@opentelemetry/instrumentation-pg": "~0.36.1",
-    "@opentelemetry/resources": "~1.17.0",
-    "@opentelemetry/sdk-trace-base": "~1.17.0",
-    "@opentelemetry/semantic-conventions": "~1.17.0",
-    "@prisma/instrumentation": "~5.3.1",
+    "@opentelemetry/core": "~1.17.1",
+    "@opentelemetry/context-async-hooks": "~1.17.1",
+    "@opentelemetry/instrumentation": "0.44.0",
```

Comment:
I hard-pinned all instrumentation dependencies, and pinned the "core" dependencies to bugfixes. LMK if you think we should hard pin _everything_ 🤔